### PR TITLE
disabling export for blank/empty files TM-1829

### PR DESCRIPTION
**Pull request by SidelineCory24**
Disables the export button when no Proxy CDO View is selected and when a CDO is selected, but the results are 0.

### CODE REVIEW

File: src/Containers/BidderPortfolio/BidderPortfolio.jsx [hunk 1]
```diff
@@ -93,8 +93,6 @@ class BidderPortfolio extends Component {
   // Form our query and then retrieve bidders.
   getBidderPortfolio(removePageandLimit) {
     const query = this.createSearchQuery(removePageandLimit);
-    // eslint-disable-next-line no-console
-    console.log('for final check query: ', query);
```
Comment (elizabeth-jimenez) [inline src/Containers/BidderPortfolio/BidderPortfolio.jsx:97]: Haha thanks!

**Comment by elizabeth-jimenez** [comment]
There is a bit of optimization we can do; we can discuss on our Thursday Dev Chat. But the functionality is there!

File: src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx [hunk 2]
```diff
@@ -66,6 +66,12 @@ class BidderPortfolioPage extends Component {
     const showEdit = editType.show;
 
     const hideControls = get(bidderPortfolio, 'results', []).length === 0 || !cdosLength;
+
+    let disableLink = false;
+    const total = get(bidderPortfolio, 'count');
+    if (cdosLength === 0 || total === 0) {
+      disableLink = true;
+    }
```
Comment (elizabeth-jimenez) [inline src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx:74]: Discussed in Discord. Just dropping here for reference:
The code above is the same as:
```
let disableLink;
const total = get(bidderPortfolio, 'count');
if (cdosLength === 0 || total === 0) {
    disableLink = true;
}
else {
    disableLink = false;
}

```
and the same as:
```
const total = get(bidderPortfolio, 'count');
let disableLink = (cdosLength === 0 || total === 0);
```

🦜

Comment (SidelineCory24) [reply src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx]: Does `const` make more sense vs. `let` after the change?

File: src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx [hunk 3]
```diff
@@ -67,11 +67,8 @@ class BidderPortfolioPage extends Component {
 
     const hideControls = get(bidderPortfolio, 'results', []).length === 0 || !cdosLength;
 
-    let disableLink = false;
     const total = get(bidderPortfolio, 'count');
-    if (cdosLength === 0 || total === 0) {
-      disableLink = true;
-    }
+    const disableLink = (cdosLength === 0 || total === 0);
```
Comment (elizabeth-jimenez) [inline src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx:71]: 🥳

File: src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx (hunk 2 above)
Comment (elizabeth-jimenez) [reply src/Components/BidderPortfolio/BidderPortfolioPage/BidderPortfolioPage.jsx]: Yes, good catch!

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍬 🥮 👍

**Pull request merged**